models/ObjectModel.js

```diff
@@ -55,7 +55,9 @@ function ObjectModel(ip, version, protocol, objectId) {
         width: 0.3, // default size should always be overridden, but exists in case xml doesn't contain size
         height: 0.3
     };
-    this.isWorldObject = false;
+    this.isWorldObject = false; // a bit redundant with this.type, but good for backwards compatibility
```

Comment:
This is a bit overly fancy but if you rewrite this as a getter/setter pair (<https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get>) you won't have to worry about it getting out of sync with this.type

Reply:
(might only need a getter)